package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vuejs.org",
   "private": true,
   "hexo": {
-    "version": "3.8.0"
+    "version": "3.9.0"
```

Comment:
Can you reverse the changes made to this file?

Reply:
Sure - I've reverted this